src/ray/protobuf/node_manager.proto

```diff
@@ -399,6 +399,13 @@ message IsLocalWorkerDeadReply {
   bool is_dead = 1;
 }
 
+message GetDriverAndWorkerPidsRequest {}
+
+message GetDriverAndWorkerPidsReply {
+  // PIDs of all drivers and workers managed by the local raylet.
+  repeated uint32 pids = 1;
+}
```

Comment:
We can just call this `GetWorkerPIDs` -- a driver is considered a worker. You can mention in a header comment that it includes the driver for completeness.